pkg/apply/applier.go

```diff
@@ -286,9 +256,9 @@ type Options struct {
 
 // setDefaults set the options to the default values if they
 // have not been provided.
-func setDefaults(o *Options) {
-	if o.PollInterval == time.Duration(0) {
-		o.PollInterval = poller.DefaultPollInterval
+func setDefaults(o *ApplierOptions) {
+	if o.PollInterval == 0 {
+		o.PollInterval = defaultPollInterval
```

Comment:
`poller.DefaultPollInterval` is not used anywhere else, so brought it here, made private and dropped the dependency on the `poller` package.